src/app.ts

```diff
@@ -193,11 +193,10 @@ async function start () {
         error('Please provide a configuration file path using -p', {});
     } else if (statusCode == 500) {
         error('Some error occurred, see logs for details', {});
-    } else {
-        app.listen(servicePort, async () => {
-            setInterval(() => prometheusChecks(pathRefreshFile), prometheusInterval);
-        });
     }
+    app.listen(servicePort, async () => {
```

Comment:
Please add a comment explaining that you want to start up even if the initial scrape is only successful in part, because I was (once more) tempted to request a change to the contrary.